packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js

```diff
@@ -480,6 +480,71 @@ const Conversation = WebexPlugin.extend({
       });
   },
 
+  /**
+   * Paginates through a set of conversations. By default does not fetch activities or
+   * participants
+   * @param {Object} options
+   * @param {boolean} options.deferDecrypt - when true, deferDecrypt tells the
+   * payload transformer to normalize (but not decrypt) each received
+   * conversation. Instead, the received conversations will each have a bound
+   * decrypt method that can be executed at the consumer's leisure
+   * @param {Page} options.page - After the first result has been returned to a consumer,
+   * you can pass the Page back to the sdk to get the next list of results.
+   * @returns {Promise<Array<Conversation>>}
+   */
+  async paginate(options = {}) {
+    if (options.page) {
+      // We were passed a page but we are out of results
+      if (!options.page.links || !options.page.links.next) {
+        throw new Error('No link to follow for the provided page');
+      }
+
+      // Go get the next page of results
+      return this.request({
+        url: options.page.links.next
+      }).then((res) => ({page: new Page(res, this.webex)}));
```

Comment:
I see all of the other requests that are utilizing `Page` return it like this:

`.then((res) => new Page(res, this.webex));`

Is this because our options object is looking for the `page` property?

Reply:
Yes. And for Federation we want to return a top level `additionalUrls` property. I didn't really want to muddy that argument with the page result as it's specific to conversation federation